bitnami/kubernetes-event-exporter/values.yaml

```diff
@@ -90,6 +90,9 @@ image:
 ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
 ##
 hostAliases: []
+## @param config.kubeQPS Queries per second limit
+## @param config.kubeBurst Up to x number of queries allowed. Burst to roughly match your events per minute QPS to be 1/5 of the burst
+## @param config.maxEventAgeSeconds maximum age of an event that can be processed
```

Comment:
The `@param` is a reserved word used by the [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm/) to associate the values with their description/metadata.

These values would be removed so this section would need to be replaced.

I would suggest a more generic approach by referencing the upstream documentation and mentioning that this setting is rendered as a template to improve the user experience:

```suggestion
## Kubernetes event exporter configuration, rendered as a template
## ref: https://github.com/resmoio/kubernetes-event-exporter#configuration
```